src/main/java/com/exactpro/th2/inframgr/repository/RepositoryWatcherService.java

```diff
@@ -132,9 +138,20 @@ private void removeExtinctedNamespaces(Set<String> existingBranches) {
             if (namespaceResource != null) {
                 String branchName = extinctNamespace.substring(namespacePrefix.length());
                 eventRouter.removeEventsForSchema(branchName);
```

Comment:
Do we need to remove events in this case? Shouldn't we apply the latest changes we have in memory?

Reply:
SchemaEventRouter is internal entity to process events. the removeEventsForSchema method removes all events related to schema from cache